stdlib/private/SwiftPrivateLibcExtras/SwiftPrivateLibcExtras.swift

```diff
@@ -97,12 +97,21 @@ public func _stdlib_select(
         let readAddr = readfds.baseAddress
         let writeAddr = writefds.baseAddress
         let errorAddr = errorfds.baseAddress
+#if CYGWIN
+        func asFdSetPtr(
+          _ p: UnsafeMutablePointer<UInt>?
+        ) -> UnsafeMutablePointer<_types_fd_set>? {
```

Comment:
Sure.  The clang-importer will ignore the `#define` based aliasing as its not a real type alias, but rather a token replacement via the preprocessor.  The result is that the module does not contain a definition for `fd_set` although, we should.  We could handle simple token replacements which replace type aliased names with an additional type alias.  This would allow us to use the definition.

After writing out this explanation, Im left wondering, why don't we work around this for now with:
  typealias fd_set = _types_fd_set